client/Lykke.Service.PayInternal.Client/Models/Transfer/TransactionRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Lykke.Service.PayInternal.Client.Models.Transfer
+{
+    public class TransactionRequest 
```

Comment:
Why do we need hash and countConfirm in request?

Reply:
One of the core ideas in legacy code for transfers is: if we get the object of some type in, we need to give out the object of the same type but with the updated inner state. I'm not sure this is a perfect decision, but it will take too much time to refactor such a logic in general.

Let's assume this is the Core Comment #1, for I'll have to mention it for some other aspects below.